modules/ossm-vs-istio.adoc

```diff
@@ -106,7 +106,7 @@ A {ProductName} control plane component called Istio OpenShift Routing (IOR) syn
 
 [id="ossm-catch-all-domains_{context}"]
 === Catch-all domains
-Catch-all domains ("\*") are not supported. If one is found in the Gateway definition, {ProductName} _will_ create the route, but will rely on OpenShift to create a default hostname. This means that the newly created route will __not__ be a catch all ("*") route, instead it will have a hostname in the form `<route-name>[-<project>].<suffix>`. Refer to the OpenShift documentation for more information about how default hostnames work and how a cluster administrator can customize it.
+Catch-all domains ("\*") are not supported. If one is found in the Gateway definition, {ProductName} _will_ create the route, but will rely on OpenShift to create a default hostname. This means that the newly created route will __not__ be a catch all ("*") route, instead it will have a hostname in the form `<route-name>[-<project>].<suffix>`. Refer to the OpenShift documentation for more information about how default hostnames work and how a `cluster-admin` can customize it. If you are using {product-dedicated}, refer to the {product-dedicated} the `dedicated-admin` role.
```

Comment:
I don't expect you to remove the future tense and markup from the para, but I sure hope someone does.